nio/router/base.py

```diff
@@ -354,15 +354,15 @@ def notify_signals(self, block, signals, output_id):
         elif self.status.is_set(RunnerStatus.stopped):
             self.logger.warning("Block Router is stopped, discarding signal"
                                 " notification from block: {}".
-                                format(block.name()))
+                                format(block.id()))
```

Comment:
Use `block.label()` in all of these warning and exception messages instead of `block.id()`